ios/Podfile.lock

```diff
@@ -7,11 +9,14 @@ PODS:
     - boost-for-react-native
     - DoubleConversion
     - glog
+  - FSCalendar (2.8.0)
   - glog (0.3.5)
   - React (0.57.7):
     - React/Core (= 0.57.7)
   - react-native-maps (0.23.0):
```

Comment:
This bit
```
  - react-native-maps (0.23.0):
    - React
```

Should be below
```
   - react-native-bpk-component-calendar (0.0.1):
     - React
```
instead of above.

That's all that's wrong with this podfile 😉